timetable/Objects/Days.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+namespace Timetable.timetable.Objects
+{
+	/// <summary>
+    /// Days.
+    /// </summary>
+	public enum Days
```

Comment:
Is ok to define this in its own file, but if you would only ever use it in DaysList, we might just put in that class.

Reply:
It is used more than once ;)